sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js

```diff
@@ -0,0 +1,484 @@
+    profiles: [{
+    thumbnail: 'Mary-Hensley-Blinn-Lois-Elfman.jpg',
+    name: 'Blinn College',
+    body: 'Part of the Blinn College District, Blinn College is a public two-year institution in Brenham, Texas. More than a century old, Blinn boasts the highest transfer rate (49%) to four-year institutions in Texas. Blinn students go on to attend schools such as Texas A&M University, the University of Texas, and the University of Houston. “This honor symbolizes Blinn’s continued commitment to its core values, including collaboration, diversity, excellence, and respect,” says Chancellor Dr. Mary Hensley. “We are honored to accept this recognition and to continue to provide faculty, staff, and students a welcoming environment where they can grow and achieve.”',
+    },
+    {
+    thumbnail: 'Gregory-Haile-Broward-Lois-Elfman.jpg',
+    name: 'Broward College',
+    body: 'In Southeast Florida and part of the Florida College System, Broward College is a public institution established in 1959. Under its current name since 2008, the college has three campuses and additional partnerships around Broward County. It serves approximately 63,000 students annually with nearly 5,000 part-time and full-time faculty and staff. “Broward College and the work we do is a vital pathway for advancing people from all backgrounds,” says President Gregory Adam Haile. “Our commitment is to continue to attract the best and brightest professionals eager to improve the lives of our residents through post-secondary education.”',
+    },
+    {
+    thumbnail: 'Misty-Hatfield-Central-Carolina-TC-Lois Elfman.jpg',
```

Comment:
Ending is .JPG

Reply:
I believe this one may be due to the space between Lois and Elfman.